consensus/fast-sync/src/fast_sync.rs

```diff
@@ -6,8 +6,6 @@ use std::{
     task::{Context, Poll},
 };
 
-#[allow(unused_imports)]
-use hex_literal::hex;
```

Comment:
This is used in the `hashes_of_hashes` file but `clippy` can't detect external files like this.